test/libsinsp_e2e/sys_call_test.cpp

```diff
@@ -505,7 +506,8 @@ TEST_F(sys_call_test, brk)
 
 		if (type == PPME_SYSCALL_BRK_4_E)
 		{
-			uint64_t addr = *((uint64_t*)e->get_param_by_name("addr")->m_val);
+			uint64_t addr;
+			memcpy(&addr,e->get_param_by_name("addr")->m_val, sizeof(uint64_t));
```

Comment:
Thanks for these fixese, but it's even easier than this!

```c++
e->get_param_by_name("addr")->as<uint64_t>();
```

😎

Reply:
I totally forgot about that! Thanks!